homeassistant/components/myuplink/icons.json

```diff
@@ -0,0 +1,45 @@
+{
+  "entity": {
+    "binary_sensor": {
+      "elect_add": {
+        "default": "mdi:electric-switch",
+        "state": {
+          "on": "mdi:electric-switch-closed"
+        }
+      }
+    },
+    "number": {
+      "degree_minutes": {
+        "default": "mdi:thermometer-lines"
+      }
+    },
+    "sensor": {
+      "air_flow": {
+        "default": "mdi:weather-windy"
+      },
+      "elect_add": {
+        "default": "mdi:heat_wave"
```

Comment:
```suggestion
        "default": "mdi:heat-wave"
```

Reply:
Good catch! Not the first time that I put an underscore in icon names.